sssd_test_framework/utils/sssd.py

```diff
@@ -783,3 +783,54 @@ def autofs(self) -> None:
         """
         self.sssd.authselect.select("sssd")
         self.sssd.enable_responder("autofs")
+
+    def proxy(self,
+              host: MultihostHost,
+              domain: str = None,
+              proxy: str = None,
+              provider: str = None,
```

Comment:
let's make the type `str | list[str]` and default to "id". So it either set one provider or multiple providers.